ipaclient/install/client.py

```diff
@@ -3457,12 +3458,20 @@ class ClientInstallInterface(hostname_.HostNameInstallInterface,
     no_ntp = enroll_only(no_ntp)
 
     force_ntpd = knob(
-        None,
+        None, False,
         description="Stop and disable any time&date synchronization services "
-                    "besides ntpd",
+                    "besides ntpd.\n"
+                    "This option has been obsoleted by --force-chrony",
```

Comment:
`deprecated=True`, otherwise it IMHO appears in `--help`.

Update man.

Reply:
Allrighty :+1: